tutorial/tutorial-gabc-01.html

```diff
@@ -268,7 +268,7 @@
 </pre>
 </div>
 
-<p>Here's the result after processing by gregorio and LaTeX:</p>
+<p>Here's the result after processing by gregorio and <span class="latex">L<span class="alpha">a</span>T<span class="epsilon">e</span>X</span>:</p>
```

Comment:
There is now a better way to capitalize the "y", but perhaps the TeX for that is too advanced for this tutorial.